### PR TITLE
feat: add keys support for sliders

### LINKED ISSUE #572: Slider component keyboard support
http://localhost:8080/#/demo/vuestic-theme/vuestic-components/va-slider/VaSlider.demo.vue
Component should be operable from keyboard.

### PR DESCRIPTION
**Pull request by smellyshovel**
## Description

Close #572 
## Markup:

<details>

```vue
// Your code
```
</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: src/vuestic-theme/vuestic-components/va-slider/VaSlider.vue [hunk 1]
```diff
@@ -330,6 +335,24 @@ export default {
         this.flag = false
       }
     },
+    moveWithKeys (event) {
+      // don't do anything if a dot isn't focused of if the slider's disabled
```
Comment (jean-moldovan) [inline src/vuestic-theme/vuestic-components/va-slider/VaSlider.vue]: typo: don't do anything if a dot isn't focused **of** if the slider's disabled

File: src/vuestic-theme/vuestic-components/va-slider/VaSlider.vue [hunk 2]
```diff
@@ -45,6 +45,7 @@
           class="va-slider__container__handler"
```
Comment (asvae) [inline src/vuestic-theme/vuestic-components/va-slider/VaSlider.vue]: image

Shouldn't go below zero or above max.

Comment (asvae) [inline src/vuestic-theme/vuestic-components/va-slider/VaSlider.vue]: Selection should be more subtle.
You can check checkbox example

http://localhost:8080/#/demo/vuestic-theme/vuestic-components/va-checkbox/VaCheckbox.demo.vue

Comment (asvae) [inline src/vuestic-theme/vuestic-components/va-slider/VaSlider.vue]: Visible selection on sliding element shouldn't trigger on click. That's already done for checkbox.

**Pull request merged**